lib/preload.js

```diff
@@ -655,6 +655,9 @@ class Preloader extends EventEmitter {
 
   _getAppData () {
     if (this._supervisorLT7()) {
+      if (this.pinDevice === true) {
+        console.warn('\nWarning: Pinning releases only works with supervisor versions >= 7.0.0')
```

Comment:
Why not fail completely?

Reply:
I didn't think of it, you're right, that's a better idea, updated.